app/main.js

```diff
@@ -21,8 +22,18 @@
         "URL_LAUNCHER_OVERLAY_SCROLLBARS": process.env.URL_LAUNCHER_CONSOLE == null ? 0 : process.env.URL_LAUNCHER_CONSOLE === '1'
     };
 
-    let window = null;
+    if (process.env.NODE_ENV === 'development') {
+      console.log('Running in development mode')
+      electronConfig = Object.assign(electronConfig, {
+        "URL_LAUNCHER_URL": 'file:///' + path.join(__dirname, 'data', 'index.html'),
+        "URL_LAUNCHER_HEIGHT": 600,
```

Comment:
Let's drop the unnecessary quotes. I'll merge https://github.com/resin-io/resin-electronjs/pull/32, you can just rebase or merge and `npm run lint` to have some sane code style.